src/poggit/errdoc/InternalErrorPage.php

```diff
@@ -64,7 +64,18 @@ public function output() {
           <code class="code"><?= htmlspecialchars($_REQUEST["id"] ?? Meta::getRequestId(), ENT_QUOTES, 'UTF-8') ?></code></p>
         <p>Logging out may solve the problem.
           <span class="action" onclick="location.assign('<?= Meta::root() ?>logout')">Have a try</span></p>
-        <a class="twitter-timeline" data-width="350" data-height="600" data-theme="light" data-link-color="#E81C4F"
+        <a class="twitter-timeline" data-width="350" data-height="600" data-theme="<?php
+        try {
+            $session = Session::getInstance();
+            Meta::getLog()->e($session->isLoggedIn());
```

Comment:
Are you sure you didn't forget to delete this line?

Reply:
My bad.